package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "bootstrap": "^4.4.1",
     "mobile-detect": "^1.4.4",
+    "react-app-polyfill": "^1.0.6",
```

Comment:
This file shouldn't be needed since browser support is determined by analytics of each site.